### PR TITLE
minor change for MUMPS 5 release

### PR DESCRIPTION
**Pull request by jordiblasco**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyblocks/m/mumps.py [hunk 1]
```diff
@@ -34,12 +34,13 @@
...
 import os
 import shutil
+from distutils.version import LooseVersion
 
 import easybuild.tools.toolchain as toolchain
 from easybuild.easyblocks.generic.configuremake import ConfigureMake
 from easybuild.tools import toolchain
 from easybuild.tools.build_log import EasyBuildError
-from easybuild.tools.modules import get_software_root
+from easybuild.tools.modules import get_software_root, get_software_version
```
Comment (boegel) [inline easybuild/easyblocks/m/mumps.py]: unused extra import?

File: easybuild/easyblocks/m/mumps.py [hunk 2]
```diff
@@ -61,7 +62,11 @@ def configure_step(self):
             optf = "-Dintel_ -DALLOW_NON_INIT -nofor-main"
             optl = "-nofor-main"
         elif comp_fam == toolchain.GCC:  #@UndefinedVariable
-            make_inc_templ = 'Makefile.gfortran.%s'
+            if LooseVersion(self.version) >= LooseVersion("5.0.0"):
+                make_inc_templ = 'Makefile.debian.%s'
```
Comment (boegel) [inline easybuild/easyblocks/m/mumps.py]: hmm, why debian? what are the other alternatives? anything more generic? `Makefile.linux` maybe?

Comment (jordiblasco) [inline easybuild/easyblocks/m/mumps.py]: Apparently MUMPS have decided to change the name of the makefile templates and Makefile.gfortran is not longer in there. The equivalent is Makefile.debian

File: easybuild/easyblocks/m/mumps.py (hunk 1 above)
Comment (boegel) [inline easybuild/easyblocks/m/mumps.py]: please merge https://github.com/jordiblasco/easybuild-easyblocks/pull/1 to fix this

File: easybuild/easyblocks/m/mumps.py (hunk 2 above)
Comment (boegel) [inline easybuild/easyblocks/m/mumps.py]: OK, weird renaming, but fine, it is what it is. :-)

**Comment by boegel** [comment]
This needs https://github.com/jordiblasco/easybuild-easyblocks/pull/1

**Comment by boegel** [comment]
verified with https://github.com/hpcugent/easybuild-easyconfigs/pull/1867, works as intended

**Comment by boegel** [comment]
still on hold to get https://github.com/jordiblasco/easybuild-easyblocks/pull/1 included

**Comment by boegel** [comment]
will be merged via #686

**Pull request merged**